apps/AppWithWearable/lib/pages/chat/page.dart

```diff
@@ -181,6 +181,7 @@ class _ChatPageState extends State<ChatPage> with AutomaticKeepAliveClientMixin
     );
   }
 
+  // FIXME: only supports 1 active plugin. When multiple are active uses the first one
```

Comment:
nice, is there a way we could have a selector on the chat page so people can select which plugin to chat, maybe like a floating dropdown top right or smth

Reply:
I don't get why more than one plugin can be active at a time. It makes more sense to only allow 1 plugin active. Also for the memories. Is there a reason several plugins can be active simultaneously?

Reply:
Ideally plugins are used only whenever they are useful, so if you are having a medical conversation, it uses x plugin, entrepreneurship, but that requires better plugin prompt, maybe you could take a look so we filter out useless plugins per memory. It's complex to expect the user will remind before every conversation what plugin they might need, so if you activate many you are set and forget

Reply:
btw, how can I find you on discord? would love to chat

Reply:
will dm

Reply:
for the multiple plugin case. We can pass all personalities to the system prompt and ask the LLM to choose which one is most relevant for the current conversation. This will fulfill the promise to the user they don't have to remember turning plugins on and off. They choose what they want, and chat layer is flexible enough to understand and adapt. Reduces strain on what is asked by us from the user improving overall UX.

I still think we should limit how many plugins are active at the same time. The more active plugins the more chances of this failing/confusing the LLM. As a measure for now we can limit active plugins to 5.